src/main/java/step2/domain/Prize.java

```diff
@@ -0,0 +1,63 @@
+package step2.domain;
+
+import java.util.Map;
+
+public enum Prize {
+    FOURTH(3, 5_000),
+    THIRD(4, 50_000),
+    SECOND(5, 1_500_000),
+    FIRST(6, 2_000_000_000);
+
+    private final int matchNumber;
+    private final int prizeMoney;
+
+    Prize(int matchNumber, int prizeMoney){
+        this.matchNumber = matchNumber;
+        this.prizeMoney = prizeMoney;
+    }
+
+    public int getMatchNumber(){
+        return matchNumber;
+    }
+
+    public int getPrizeMoney(){
+        return prizeMoney;
+    }
+
+    public static int getPrize(int matchNumber){
+        if(matchNumber==Prize.FIRST.matchNumber){
+            return Prize.FIRST.getPrizeMoney();
+        }
+        if(matchNumber==Prize.SECOND.matchNumber){
+            return Prize.SECOND.getPrizeMoney();
+        }
+        if(matchNumber==Prize.THIRD.matchNumber){
+            return Prize.THIRD.getPrizeMoney();
+        }
+        if(matchNumber==Prize.FOURTH.matchNumber){
+            return Prize.FOURTH.getPrizeMoney();
+        }
+        return 0;
+    }
+
+    public static int calculateSumOfPrizeMoney(Map<Integer, Integer> result){
+        int sumOfPrizeMoney = 0;
+        if(result.get(Prize.FOURTH.getMatchNumber())>0) {
+            sumOfPrizeMoney += result.get(Prize.FOURTH.getMatchNumber()) * Prize.FOURTH.getPrizeMoney();
+        }
+        if(result.get(Prize.THIRD.getMatchNumber())>0) {
```

Comment:
```suggestion
        if (result.get(Prize.THIRD.getMatchNumber()) > 0) {
```
IDE 에서 제공하는 코드 포멧팅을 활용해 봅시다.